org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/linkedediting/LSPLinkedEditingReconcilingStrategy.java

```diff
@@ -59,6 +59,7 @@ public class LSPLinkedEditingReconcilingStrategy extends LSPLinkedEditingBase
 	private @Nullable EditorSelectionChangedListener editorSelectionChangedListener;
 	private @Nullable Job highlightJob;
 	private @Nullable LinkedModeModel linkedModel;
+	private int lastSelectionLength;
```

Comment:
This field now seems useless. Can you please remove it?

Reply:
Done. I was still working on a test.